datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -369,6 +454,29 @@ fn summarize_min_max(
                     .unwrap_or_else(|_| min_values[i] = None);
             }
         }
+
+        ParquetStatistics::ByteArray(s)
+            if matches!(fields[i].data_type(), DataType::Utf8 | DataType::LargeUtf8) =>
+        {
+            if let Some(max_value) = &mut max_values[i] {
```

Comment:
I believe byte arrays are also used to store `DataType::Decimal` values as well (though hopefully if we consolidate the statistics conversion code it will "just work")